src/sound.c

```diff
@@ -1,6 +1,6 @@
-/* vi:set ts=8 sts=4 sw=4 noet:
+/* vi:set ts=4 sts=4 sw=4 et:
```

Comment:
Why change 'ts' when 'sts' is already set?

Reply:
For `retab` and for clarity. I think make more sense to have all 4.

Reply:
How? For `:retab` a Tab should be expanded to 8 spaces, not 4.

Reply:
I've changed it